docs/cost-management/README.md

```diff
@@ -0,0 +1,201 @@
+   ```
+1. Return to the [console.redhat.com](https://console.redhat.com/) **Sources** wizard, enter your **Subscription ID**. Click **Next** to move to the next screen
+
+1. In Azure CLI, create a cost management Storage Account Contributor role, an obtain your **tenant ID**, **client (application) ID**, and **client secret**
+
+    ```bash
+    az ad sp create-for-rbac -n "CostManagement" --role "Storage Account Contributor" --query '{"tenant": tenant, "client_id": appId, "secret": password}'
+    ```
+1. Return to **Sources** wizard in [console.redhat.com](https://console.redhat.com/), enter your Azure **Tenant ID**, **Client ID**, and **Client Secret**.   
+
+1. Run the following command to create cost management Reader role with your subscription ID. Copy the full command from the **Sources** wizard, which will automatically substitute your Azure subscription ID obtained earlier. 
+
+   ```bash
+   az role assignment create --role "Cost Management Reader" --assignee http://CostManagement --subscription <SubscriptionID>
+   ```
+1. Click **Next** in **Sources** wizard.
+
+
+### 2.3 Configuring a Daily Azure data export schedule using Azure Portal
+
+*Cost management requires a data export from a **Subscription** level scope*
+
+1. In the Azure Portal home page, click on **Subscriptions**
+<img src="images/subscriptionpg.png"> 
+
+1. Select the Subscription you want to track from the list, and then select **Cost Analysis** in the menu. At the top of the **Cost analysis** page, select **configure subscription**
+<img src="images/cost-analysis.png"> 
+
+1. Click on the **Export** tab, and then **Schedule export**
+<img src="images/schedule-export.png"> 
+
+1. In the **Exports** wizard, fill out the **Export details** 
+   
+   <img src="images/new-exports.png"> 
+   
+   * For **Export Type**, select **Daily export of billing-period-to-date costs** <br>
+
+   * For **Storage account**, select the account you created earlier <br>
+
+   * Enter any value for the **Container name** and **Directory path** for the export. These values provide the tree structure in the storage account where report files are stored.<br>
+
+   * Click **Create** to start exporting data to the Azure storage container. 
+
+1. Return to **Sources** wizard after creating the export schedule and click **Next**. Review the source details 
+
+1. Click **Finish** to complete adding the Azure source to cost management
+
+Cost management will begin polling Azure for cost data, which will appear on the cost management dashboard (console.redhat.com/openshift/cost-management/).
+
+
+## Managing your Costs
+
+After adding your Openshift Container Platform and Cloud Provider sources, Cost management will show cost data by 
+1. Source
+1. Cloud provider cost and usage related to running your OpenShift Container Platform clusters on their platform 
+
+> See the following video for a quick overview of Cost Management for OpenShift followed by a demo of the product
```

Comment:
this should be indented three times `   > See the ....`  in order to be inside the ordered list.